integration-tests/src/test/java/org/apache/fineract/integrationtests/DatatableIntegrationTest.java

```diff
@@ -247,6 +252,15 @@ public void validateCreateReadDeleteDatatable() {
         assertEquals(datatableEntryMap.get("itsAString"), ((Map) updatedDatatableEntryResponse.get("changes")).get("itsAString"));
         assertEquals(datatableEntryMap.get("itsAText"), ((Map) updatedDatatableEntryResponse.get("changes")).get("itsAText"));
 
+        FineractClient fineract = FineractClient.builder().insecure(true).baseURL("https://localhost:8443/fineract-provider/api/v1/")
+                .tenant("default").basicAuth("mifos", "password").build();
+        String rows = Calls.ok(fineract.dataTables.queryValues(datatableName, "itsABoolean",
```

Comment:
Please call a ".toLowerCase" on the "datatableName" variable. In postgres unfortunately the lookup not do it automatically.

Reply:
fixed